Makefile

```diff
@@ -26,3 +26,32 @@ envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
+GOLANGCI ?= $(LOCALBIN)/golangci-lint
+
+.PHONY: golangci
+golangci: $(GOLANGCI) ## Download golangci-lint locally if necessary.
+$(GOLANGCI): $(LOCALBIN)
```

Comment:
here and L43-46: I do not really know what this is doing. I copied this from envtest but it seems to work as expected